src/rpc/CMakeLists.txt

```diff
@@ -29,25 +29,122 @@
 set(rpc_sources
   core_rpc_server.cpp)
 
+set(daemon_messages_sources
+  "message.cpp"
```

Comment:
I don't think there is one, save for filenames with whitespace (which I'd prefer to avoid anyway).  Not sure why I used quotes here.